packages/devtools_app/lib/src/inspector/flutter/inspector_data_models.dart

```diff
@@ -421,17 +542,23 @@ class FlexLayoutProperties extends LayoutProperties {
       EnumUtils<TextBaseline>(TextBaseline.values);
 }
 
-class RenderInfo {
-  RenderInfo(this.axis, Size size, Offset offset, Size realSize)
-      : width = size.width,
-        height = size.height,
-        realWidth = realSize.width,
-        realHeight = realSize.height,
-        dx = offset.dx,
-        dy = offset.dy;
+class RenderProps {
```

Comment:
nit: maybe call this `LayoutInfo` or `LayoutProperties`.
Avoid `Props` as it is an abbreviation and names should avoid abbreviations where possible.